cpp/ps0005/problem 5.cpp

```diff
@@ -10,42 +11,46 @@ int main() {
     cout << "Enter second number:\n";
     cin >> num2;
 
+    // checking user inputs and processing 4/4
     double result;
     switch(operation) {
         case '+':
-        result = num1 + num2;
-        break;
+            result = num1 + num2;
+            break;
+        
         case '-':
-        result = num1 - num2;
-        break;
+            result = num1 - num2;
+            break;
+        
         case '*':
-        result = num1 * num2;
-        break;
+            result = num1 * num2;
+            break;
+        
         case '/':
-        if (num2 != 0) {
-        result = num1 / num2;
-        }else {
-        cout << "Math eror cannot divide by zero \n" ;
-        return 1;
-        }
-        break;
+            if (num2 != 0) {
+                result = num1 / num2;
+            } else {
+                cout << "Math eror cannot divide by zero \n" ;
+                return 1;
+            } // 1/1
+            break;
+        
         case '%':
-        if ((num1 == (int)num1) && (num2 == (int)num2)) {
-        result = (int)num1 % (int)num2;
-        }else {
-        cout << "Cannot execute modulo operation on floating numbers\n";
-        return 1;
-        }
-        break;
-        case '^':
-        cout << "invalid character for operator \n";
-        return 1;
-        break;
+            if ((num1 == (int)num1) && (num2 == (int)num2)) {
+            result = (int)num1 % (int)num2;
+            } else {
+                cout << "Cannot execute modulo operation on floating numbers\n";
+                return 1;
+            } // 1/1
+            break;
+        
         default:
-        cout << "Invalid operation!\n";
-        return 1;
-    }
+            cout << "Invalid operation!\n";
+            return 1; // 1/1
+    } // 1/1
 
+    // printing the result if all data is valid 1/1
     cout << "Result: " << result ;
     return 0;
 }
+// great work 😎 10/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors while compilation
- [x] formatted code to be clear and readable
- [x] no bugs or warnings while execution
- [ ] commented some phrases for blocks of code
- [x] main returns non-zero/zero number according to the case
+0.5 bonus mark for your effort ⭐

you can check the [problem solution](https://onlinegdb.com/2mF7i4R8H) for more